docs/dev/object-deletion.md

```diff
@@ -0,0 +1,118 @@
+# Manifest Annotation For Object Deletion
+
+Developers can remove any of the currently managed CVO objects by modifying an existing manifest and adding the delete annotation .metadata.annotations["release.openshift.io/delete"]="true". This manifest annotation requests the CVO delete the in-cluster object instead of creating/updating it.
+
+Actual object deletion and subsequent deletion monitoring and status checking only occur during an upgrade. During initial installation the delete annotation prevents further processing of the manifest since the given object should not be created.
+
+## Implementation Details
+
+When the following annotation appears in a CVO supported manifest and is set to "true" the associated object will be removed from the cluster by the CVO.
```

Comment:
nit: "true" -> `true` to match the subsequent line.

Reply:
fixed